### DISCUSSION TITLE
Releases "XXX commits to master since this release" must be a configurable branch

### DISCUSSION THREAD

**Discussion by marekr** [community/community#68862] [Repositories]
### Select Topic Area

Product Feedback

### Body

The inability to configure the "base branch" for a release, makes this statement

![image](https://github.com/community/community/assets/1838013/1b7b3a9c-0bcf-44c6-98c6-c2b1b9bc600a)

nonsensical.

In our workflow, our releases are on a "stable" branch and the tags are made there, the master branch is indeed 3000 commits later and is considered the unstable branch until a release 1 year later. We make the unstable branch our default because that is indeed our development head and the stable is purely bug fixing.

It would be great if this was configurable on Releases.

**Comment by dereuromark** [comment]
They completely removed it :cry: That's terrible, this was one of the most useful infos here in the releases page for maintainers!! Even the default branch already helped here..

**Reply by andig** [reply]
See https://github.com/orgs/community/discussions/201116
